Source/WebCore/rendering/RenderObject.cpp

```diff
@@ -974,19 +974,22 @@ void RenderObject::propagateRepaintToParentWithOutlineAutoIfNeeded(const RenderL
     ASSERT_NOT_REACHED();
 }
 
-void RenderObject::repaintUsingContainer(const RenderLayerModelObject* repaintContainer, const LayoutRect& r, bool shouldClipToLayer) const
+void RenderObject::repaintUsingContainer(SingleThreadWeakPtr<const RenderLayerModelObject> repaintContainer, const LayoutRect& r, bool shouldClipToLayer) const
 {
     if (r.isEmpty())
         return;
 
     if (!repaintContainer)
-        repaintContainer = &view();
+        repaintContainer = { view() };
```

Comment:
You likely do not need these curly brackets.

Reply:
This change was reverted.